packages/main/src/plugin/index.ts

```diff
@@ -519,6 +531,8 @@ export class PluginSystem {
     const telemetry = container.get<Telemetry>(Telemetry);
     await telemetry.init();
 
+    this.loadEnterpriseTelemetry(container);
```

Comment:
Odd it's in initExtensions the `this.loadEnterpriseTelemetry(container);`

Should we have this somewhere else / in configuration-registry or somewhere else appropriate?